package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Social bookmarking service",
   "main": "src/backend/index.js",
   "scripts": {
-    "start": "node ./src/backend/index.js",
+    "start": "nodemon --exec babel-node ./src/backend/index.js",
```

Comment:
Leave the `start` script to run `babel-node ./src/backend/index.js"` and then create a second script eg `dev` to run `nodemon --exec 'babel-node ./src/backend/index.js'`